src/test/java/reposense/util/TestUtil.java

```diff
@@ -104,6 +104,19 @@ public static Date getDate(int year, int month, int date) {
                 .getTime();
     }
 
+    /**
+     * Creates and returns a {@code Date} object with the specified {@code year}, {@code month}, {@code day} that is
+     * at the end of the day.
+     */
+    public static Date getEndOfDate(int year, int month, int date) {
```

Comment:
I somehow remember we already got a smililar method in our [code base](https://github.com/reposense/RepoSense/blob/master/src/main/java/reposense/parser/ArgsParser.java#L232)? Can consider extracting all date related method to a `DateUtil` or `TimeUtil`. But can done in a separate PR though.